src/pages/register-a-beacon/check-beacon-summary.tsx

```diff
@@ -56,13 +71,14 @@ const BeaconNotRegisteredView: FunctionComponent<BeaconDetailsProps> = (
   );
 };
 
-const BeaconSummary: FunctionComponent<BeaconDetailsProps> = ({
+const BeaconSummary: FunctionComponent<BeaconDetailsSummaryProps> = ({
   manufacturer,
   model,
   hexId,
-}: BeaconDetailsProps): JSX.Element => (
+  heading,
+}: BeaconDetailsSummaryProps): JSX.Element => (
   <>
-    <h1 className="govuk-heading-l">Check beacon summary</h1>
+    <h1 className="govuk-heading-l">{heading}</h1>
```

Comment:
We have a `PageHeading` component in the `Typography` file that you can replace this with.  The approach we have taken with replacing these components is to just replace them as we come to them